.github/workflows/dapr-test-sdk.yml

```diff
@@ -0,0 +1,220 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: dapr-test-sdk
+
+on:
+  # Run every 12 hours on weekdays, and every 24 hours on weekends.
+  schedule:
+    - cron: "0 */12 * * 1-5"
+    - cron: "30 0 * * 0,6"
+  issue_comment:
+    types: [created]
+env:
+  GOOS: linux
+  GOARCH: amd64
+  GOPROXY: https://proxy.golang.org
+
+# Job(s) can be triggered with the following commands:
+  # /test-sdk-all
+  # /test-sdk-python
+  # /test-sdk-java
+  # /test-sdk-go
+# Because of the way the GitHub handles string litterals, we need to use
+# `fromJSON` so that we only match on commands which are on their own line
+# with no space text.
+# Janky, but this is produces better UX for maintainers and prevents accidental
+# runs.
+jobs:
+  python-sdk:
+    if: |
+      github.event_name == 'schedule' ||
+      (github.event_name == 'issue_comment' &&
+        (
+          contains(format(fromJSON('"\r\n{0}\r\n"'), github.event.comment.body), fromJSON('"\r\n/test-sdk-all\r\n"')) ||
+          contains(format(fromJSON('"\r\n{0}\r\n"'), github.event.comment.body), fromJSON('"\r\n/test-sdk-python\r\n"'))
+        )
+      )
+    name: "Python SDK verification tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+      - name: "Set up Go"
+        id: setup-go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: "go.mod"
+      - name: Checkout p repo to run tests.
+        uses: actions/checkout@v3
+        with:
+          repository: dapr/python-sdk
+          path: python-sdk
+      - name: Set up Dapr CLI
+        run: wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash -s
+      - name: Build and override daprd with HEAD.
+        run: |
+          make
+          mkdir -p $HOME/.dapr/bin/
+          cp dist/linux_amd64/release/daprd $HOME/.dapr/bin/daprd
+      - name: Initialize Dapr runtime
+        run: |
+          dapr init
+      - name: Override placement service.
+        run: |
+          docker stop dapr_placement
+          ./dist/linux_amd64/release/placement --healthz-port 9091 &
+      - name: Install dependencies
+        run: |
+          cd python-sdk
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine tox
+      - name: Check Python Examples
+        run: |
+          cd python-sdk || true
+          tox -e examples
```

Comment:
Can you add reporting on test progress as PR comment? This was done in E2E tests and it looks super nice and easy to follow progress.

Reply:
Hi @artursouza, I have updated the non-draft Java and Python PR https://github.com/dapr/dapr/pull/6304/commits/b1f496f9eaaaceb9b59690e247067452501131ea